go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/mattermost/mattermost-plugin-starter-template
 go 1.16
 
 require (
-	github.com/mattermost/mattermost-server/v6 v6.0.0-20210906125346-b41b7eae1026
+	github.com/mattermost/mattermost-server/v6 v6.2.1
```

Comment:
The min server version should be set to at least `5.37.0` since we're importing a server version greater than `5.36.0`

Reply:
Eagle eyes, Michael :+1: I've bumped the version.